packages/manager-components/src/components/guides-header/guides-header.component.tsx

```diff
@@ -60,7 +60,9 @@ export function GuidesHeader({
         <GuidesHeaderItem
           key={guide}
           guide={guides[guide]}
-          href={`${guides[guide].url[ovhSubsidiary]}`}
+          href={
+            guides[guide].url[ovhSubsidiary] || guides[guide].url["DEFAULT"]
```

Comment:
> Check failure on line 64 in packages/manager-components/src/components/guides-header/guides-header.component.tsx

GitHub Actions
/ Lint Code Base

["DEFAULT"] is better written in dot notation

Can you please fix this lint error?
```suggestion
            guides[guide].url[ovhSubsidiary] || guides[guide].url.DEFAULT
```